lib/finite_machine/observer.rb

```diff
@@ -40,11 +39,12 @@ def self.cleanup_callback_queue
     #
     # @api public
     def initialize(machine)
+      @id = SecureRandom.uuid
```

Comment:
Could you please move it down so that the `machine` is the first instance assignment?

Reply:
Done.